UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -993,7 +1000,7 @@ public void CalculateOverallHealth()
 				currentHealth -= implant.TotalDamageWithoutOxyCloneRadStam;
 			}
 
-			if (brain == null || brain.RelatedPart.Health < -100)
+			if (brain == null || brain.RelatedPart.Health < -100 || brain.RelatedPart.TotalModified == 0)
```

Comment:
is TotalModified a float? If yes, use [Mathf](https://docs.unity3d.com/ScriptReference/Mathf.html).Approximately()

Reply:
well  it get set directly to 0 
so it should be okay

Reply:
![image](https://user-images.githubusercontent.com/34368774/235369512-21b4653c-8039-4dbe-a649-15881ff5f33e.png)

never trust a float

Reply:
floats have an extension so you can just ``brain.RelatedPart.TotalModified.Approx(0)`` it is painless and will save you from issues. It has been there for a century now but you keep refusing to use it.

Reply:
because I've tested it and it works 
it passes in zero as a value 
```
consumer.PowerModifier.Multiplier = 0.0f;
```

Reply:
And as expect the cyborg dies when it runs out of power